.github/workflows/imports.yml

```diff
@@ -0,0 +1,49 @@
+# Run python scripts to import/sync data from external sources
+
+name: Data imports
+
+on:
+  workflow_dispatch:
+    inputs:
+      Task:
+        type: choice
+        description: "The task to perform"
+        required: true
+        options:
+          - 'hubspot'
```

Comment:
I reckon we don't want a worflow for hubspot and another for let's say zendesk, etc.